xgettext.js

```diff
@@ -164,7 +164,9 @@ XGettext.prototype._discoverMatches = function( parsedInput ) {
 
 	traverse( parsedInput.ast, {
 		CallExpression: function( node ) {
-			var functionName = ( node.callee.property ) ? node.callee.property.name : node.callee.name;
+			// Pull the resultingFunction out of (0, resultingFunction)()
+			var callee = ( node.callee.type === 'SequenceExpression' ) ? _.last( node.callee.expressions ) : node.callee;
```

Comment:
As a side note, it would have been nice to move this out into our i18nutils and handle it outside of xgetttext, the way we do with string concatenation, but this problem occurs too soon.

I also thought about making it optional, either with a flag or some sort of over-the-top pattern matching, but it adheres so closely to the semantics of the language that I feel like the right thing to do is just pseudo-evaluate it like this.